specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -122,6 +122,37 @@
         }
       }
     },
+    "/app/subLedgerIds": {
```

Comment:
new API should require a new api-version.

Is ConfidentialLedger on private preview or public preview?